### ISSUE TITLE
[Bug] BackgroundImageSource on ContentPage does have different behaviors on Android and iOS

### ISSUE THREAD

**Issue by acuntex** [xamarin/Xamarin.Forms#11512]
### Description
BackgroundImageSource on ContentPage does have different behaviors on Android and iOS.
On iOS the background image is repeated/tiled while on Android it is stretched.

### Steps to Reproduce

1. Add an Image to the projects.
2. Set BackgroundImageSource of ContentPage

### Expected Behavior
The platforms render the image in the same way.
If there is a PCL property all platforms should have the same behavior/result.
Either tile or stretch, but at least it should be consistent.
Alternative: Remove PCL property.

### Actual Behavior
The platforms render the image differently.

### Basic Information

- Version with issue: 4.7.0.1142
- Last known good version: -
- IDE: VS Code Mac
- Platform Target Frameworks: 
  - iOS:  13.6
  - Android: 9.0

### Screenshots
![BackgroundImageSource_iOS](https://user-images.githubusercontent.com/17462039/88209864-9a38ff80-cc53-11ea-990c-97a1f8546283.png)
![BackgroundImageSource_Android](https://user-images.githubusercontent.com/17462039/88209877-9efdb380-cc53-11ea-8f9f-7fdf38466065.png)

### Reproduction Link
https://github.com/acuntex/XamarinBug_BackgroundImageSource

**Comment by Axemasta** [comment]
I think a sensible addition would be adding a `BackgroundImageFit` property to specify the behaviour of the background image. Much like `object-fit` in css you could have:

- fill
- cover
- aspect fill
- etc

A lot of workarounds on google are horrendous by nesting the background image in a grid:
```xaml
<!-- Yuck -->
<Grid>
    <Image Source="MyImage" Aspect="Fill"/>
</Grid>
```

I currently use a custom renderer on iOS to get around this issue, I'm hella lazy so i just apply the renderer to the specific page I want a background image on, as it's only 1 page in my app.

```csharp
public class LoginPageRenderer : PageRenderer
{
    protected override void OnElementChanged(VisualElementChangedEventArgs e)
    {
        base.OnElementChanged(e);

        if (e.OldElement != null || Element == null)
        {
            return;
        }

        var page = e.NewElement as Views.LoginPage;

        if (page == null)
        {
            return;
        }

        SetBackgroundImage(page);
    }

#pragma warning disable CS0618 //BackgroundImage is obsolete
    private void SetBackgroundImage(Views.LoginPage page)
    {
        UIGraphics.BeginImageContext(View.Frame.Size);
        UIImage i = UIImage.FromBundle(page.BackgroundImage);

        if (i == null)
        {
            throw new Exception($"UIImage.FromBundle was null for BackgroundImage path: {page.BackgroundImage}");
        }

        i = i.Scale(View.Frame.Size);

        View.BackgroundColor = UIColor.FromPatternImage(i);
    }
#pragma warning restore CS0618
}
```

**Comment by c0ff33-b34n** [comment]
Yes I was using the workaround with the grid too, but now I've come to optimize performance of my app on Android and trying to eliminate as many nested layouts of different types as possible, based on advice in the official Xamarin docs. Would be nice to simply specify Aspect for the background image as you can for any regular image in Xamarin Forms.
